lib/legacy/polymer.dom.html

```diff
@@ -303,8 +303,8 @@
    * @summary Legacy DOM and Event manipulation API wrapper factory used to
    * abstract differences between native Shadow DOM and "Shady DOM."
    * @memberof Polymer
-   * @param {!Node|Event} obj Node or event to operate on
-   * @return {DomApi|EventApi} Wrapper providing either node API or event API
+   * @param {(Node|Event)=} obj Node or event to operate on
+   * @return {!DomApi|!EventApi} Wrapper providing either node API or event API
```

Comment:
Is there a way to type this in Closure to simulate TypeScript overrides? Something like: `@type {(function(Node=):!DomApi)|(function(Event=):!EventApi)}`